crates/l2/Makefile

```diff
@@ -14,7 +14,7 @@ help: ## 📚 Show help for each of the Makefile recipes
 init: init-local-l1 deploy-l1 update-system-contracts init-l2 ## 🚀 Initializes a localnet with Lambda ethrex client as both L1 and L2
 
 ## Same as init but does not do deposits for rich accounts since that doesn't make sense for deployments to devnets/testnets i.e Sepolia
-init-testnet: init-local-l1 deploy-l1-testnet update-system-contracts  init-l2
+init-testnet: deploy-l1-testnet update-system-contracts  init-l2
```

Comment:
Why `init-local-l1` wasn't necessary?

Reply:
As discussed internally, we do not want to deploy a local L1 when deploying L2 on a testnet.